canonical-web-and-design/jaas-dashboard
# Render application icons to the Topology

## Done

- Application and subordinate icons are rendered onto the Topology.
- Annotation data has been requested and stored in the model status.
- The application icons have not yet been positioned, that'll come in a follow-up.


## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View a model and 'inspect' the canvas, you should see a number of `g` elements. One for each application.

## Details

Fixes #238
